core-text/src/font.rs

```diff
@@ -539,7 +539,8 @@ extern {
     fn CTFontCreateWithFontDescriptor(descriptor: CTFontDescriptorRef, size: CGFloat,
                                       matrix: *const CGAffineTransform) -> CTFontRef;
     //fn CTFontCreateWithFontDescriptorAndOptions
-    //fn CTFontCreateUIFontForLanguage
+    #[cfg(test)]
```

Comment:
Why is this only for tests?

Reply:
I'm currently only using it tests and otherwise we get a warning about it being unused.